modules/core/shared/src/main/scala/gem/math/RadialVelocity.scala

```diff
@@ -6,15 +6,28 @@ package gem.math
 import cats.{ Order, Show }
 import cats.implicits._
 import gem.math.PhysicalConstants.SpeedOfLight
-
-/** Radial Velocity represented as integral meters per second, positive if receding. */
+import scala.math.sqrt
+
+/**
+ * Radial Velocity represented as integral meters per second, positive if receding. We can also
+ * view velocity in terms of redshift (a dimensionless value called '''z''').
+ * @see Wikipedia on [[https://en.wikipedia.org/wiki/Radial_velocity Radial Velocity]]
+ * @see Wikipedia on [[https://en.wikipedia.org/wiki/Redshift Redshift]]
+ */
 final case class RadialVelocity(toMetersPerSecond: Int) {
 
   // Sanity check
   assert(toMetersPerSecond.abs <= SpeedOfLight, "Radial velocity exceeds the speed of light.")
 
   def toKilometersPerSecond: Double =
-    toMetersPerSecond.toDouble / 1000
+    toMetersPerSecond.toDouble / 1000.0
+
+  def toRedshift: Double = {
+    val v = toMetersPerSecond.toDouble
+    val C = SpeedOfLight.toDouble
+    val t = (1.0 + v / C) / (1.0 - v / C)
+    sqrt(t) - 1.0
+  }
 
```

Comment:
The redshift conversion doesn't really need to be part of this PR but I couldn't figure out how to pull it out. In any case it's simple math taken from Carlos's implementation in OCS.